eclair-core/src/main/scala/fr/acinq/eclair/crypto/keymanager/ChannelKeyManager.scala

```diff
@@ -17,95 +17,34 @@
 package fr.acinq.eclair.crypto.keymanager
 
 import fr.acinq.bitcoin.scalacompat.Crypto.{PrivateKey, PublicKey}
-import fr.acinq.bitcoin.scalacompat.DeterministicWallet.ExtendedPublicKey
-import fr.acinq.bitcoin.scalacompat.{ByteVector64, Crypto, DeterministicWallet, OutPoint, Protocol, TxOut}
-import fr.acinq.eclair.channel.{ChannelConfig, LocalParams}
-import fr.acinq.eclair.transactions.Transactions.{CommitmentFormat, TransactionWithInputInfo, TxOwner}
-import scodec.bits.ByteVector
+import fr.acinq.bitcoin.scalacompat.{Crypto, DeterministicWallet, Protocol}
+import fr.acinq.eclair.channel.ChannelConfig
 
 import java.io.ByteArrayInputStream
 import java.nio.ByteOrder
 
 trait ChannelKeyManager {
```

Comment:
Do we really need different possible implementations now ? it could just be a concrete class